packages/wrangler/src/__tests__/pages/secret.test.ts

```diff
@@ -22,16 +25,6 @@ function createFetchResult(result: unknown, success = true) {
 	};
 }
 
-export function mockGetMemberships(
```

Comment:
just like nomralize-progress-steps, importing this function was causing this suite to be run twice. we already had this helper in /helpers/mock-oauth-flow, so I replaced it with that